lms/templates/courseware/progress.html

```diff
@@ -47,9 +47,12 @@
           <header class="progress-certificates">
             <h1 class="progress-certificates-title">${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email) | h}</h1>
           </header>
-
-          %if show_generate_cert_btn:
-          <div class="wrapper-msg wrapper-auto-cert">
+        <div class="wrapper-msg wrapper-auto-cert">
+          %if enrollment_mode == 'audit':
```

Comment:
Upon closer look, I really don't like this.
We have in https://github.com/edx/edx-platform/blob/master/common/djangoapps/course_modes/models.py#L115 a lot of constants we can use to determine whether the enrollment mode is `audit`. Please leverage those constants